RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
```

Comment:
let's make this a minor 😊

```suggestion
Release type: minor
```

Reply:
can you update the docs too? 😊 https://strawberry.rocks/docs/types/resolvers#accessing-execution-information

Reply:
done, let me know if the text (and type) makes sense in the docs